src/main/java/duke/DeleteInput.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+public class DeleteInput extends InputHandler {
+
+    public DeleteInput(Ui ui, TaskList taskList) {
+        super(ui, taskList);
+    }
+
+    @Override
+    public String handle(String input) throws EmptyDescriptionException {
+        if (input.length() == 6) {
```

Comment:
I'm not too sure whether i understand the reasoning behind using length() here. Could you comment on this?